project/BuildCommon.scala

```diff
@@ -1,4 +1,4 @@
-import com.trafficland.augmentsbt._
+import com.trafficland.augmentsbt.versionmanagement.VersionManagementPlugin.autoImport.toVersion
```

Comment:
We could put the implicit into a trait and then mix it into the package object and the plugin's autoImport. Thoughts?